client/src/components/ErrorConfig.tsx

```diff
@@ -0,0 +1,48 @@
+/*
+This software is Copyright ©️ 2020 The University of Southern California. All Rights Reserved. 
+Permission to use, copy, modify, and distribute this software and its documentation for educational, research and non-profit purposes, without fee, and without a written agreement is hereby granted, provided that the above copyright notice and subject to the full license file found in the root of this software deliverable. Permission to make commercial use of this software may be obtained by contacting:  USC Stevens Center for Innovation University of Southern California 1150 S. Olive Street, Suite 2300, Los Angeles, CA 90115, USA Email: accounting@stevens.usc.edu
+
+The full terms of this copyright and license should always be found in the root directory of this software deliverable as "license.txt" and if these terms are not found with this software, please contact the USC Stevens Center for the full license.
+*/
+import React from "react";
+import { ErrorConfig } from "./types";
+export function errorForStatus(status: number): ErrorConfig {
+  switch (status) {
+    case 400:
+      return {
+        title: "Lesson not provided",
+        message:
+          "A lesson id was not provided in the url. Please contact the creator of this lesson to obtain a new link.",
+        buttonText: "OK",
+      };
+    case 404:
+      return {
+        title: "Lesson not found",
+        message:
+          "This lesson does not exist in the OpenTutor system. Please go back and try again, or contact your teacher for help.",
+        buttonText: "OK",
+      };
+    case 403:
+      return {
+        title: "Nice Try!",
```

Comment:
Let's soften this up and not explicitly accuse people of cheating (because if the problem is a system error it would be really off putting)

How about 

```typescript
{
  title: "Session Validation Failed",
  message": "We're unable to validate your session"
}
```

Reply:
How about saying "Did you think we wouldn't know you tried to edit the metadata?"

Reply:
Or I guess we could remove the sass

Reply:
yeah, i don't mind the sass if we could be SURE people are cheating, but I think just in case this gets triggered by some error in the system (a bad deploy) we should err on the side of not adding insult to injury

Reply:
I asked Rush, a 403 only happens if the sessionInfo is tampered with

Reply:
Up to you

Reply:
yeah, let's err on the side of caution and not explicitly accuse the user of cheating

Reply:
:(